problem_builder/mixins.py

```diff
@@ -266,28 +263,26 @@ def delete_key(self, dictionary, key):
         return dictionary
 
 
-class MakeURLAbsoluteMixin(object):
-    url_pattren = re.compile('<img\s*src="(.*)"')
+class ExpandStaticURLMixin(object):
 
-    def make_url_absolute(self, text):
+    def expand_static_url(self, text):
```

Comment:
Nit: should be `expand_static_urls` since this scans text and replaces multiple URL occurrences in the supplied `text`, unlike the dndv2 code for `expand_static_url` which accepts a single URL string.